lucida/commandcenter/app.py

```diff
@@ -1,17 +1,18 @@
 from __future__ import absolute_import
 from __future__ import division
 from __future__ import unicode_literals
- 
-import sys, glob, os 
-sys.path.insert(0, glob.glob(os.path.abspath(os.path.dirname(__file__)) + 
-	'/../../tools/thrift-0.9.3/lib/py/build/lib*')[0])
- 
+
+import sys, glob, os
+sys.path.insert(0, glob.glob(os.path.abspath(os.path.dirname(__file__)) +
+    '/../../tools/thrift-0.9.3/lib/py/build/lib*')[0])
+
 from controllers import *
 from flask import *
 from threading import Thread
 import logging
 
 
+
```

Comment:
I'll remove all of these extra white spaces. I stupidly diff -w to ignore the whitespace.